stdlib/2/subprocess.pyi

```diff
@@ -57,10 +57,15 @@ STDOUT = ...  # type: int
 
 class CalledProcessError(Exception):
     returncode = 0
-    cmd = ...  # type: str
-    output = ...  # type: str  # May be None
+    # morally: Union[str, bytes, List[str], List[bytes]]
```

Comment:
str and bytes are the same thing in Python 2. You should use str and unicode, or bytes and Text if you want the same code in 2 and 3. (Maybe we can merge these stubs in the future?)